wiki/extensions/physics/box2d.md

```diff
@@ -64,10 +64,12 @@ Box2DDebugRenderer debugRenderer = new Box2DDebugRenderer();
 To update our simulation we need to tell our world to step. Stepping basically updates the world objects through time. The best place to call our step function is at the end of our `render()` loop. In a perfect world everyone's frame rate is the same
 
 ```java
-world.step(1/60f, 6, 2);
+world.step(1/120f, 6, 2);
 ```
 
-The first argument is the time-step, or the amount of time you want your world to simulate. In most cases you want this to be a fixed time step. libGDX recommends using a value between `1/60f` (which is 1/60th of a second) and `1/240f` (1/240th of a second).
+The first argument is the time-step, or the amount of time you want your world to simulate. In most cases you want this to be a fixed time step. libGDX recommends using a value between `1/120f` (which is 1/120th of a second) and `1/240f` (1/240th of a second).
+
+In general, you should always at least use a rate which is twice as your FPS (and so your time step should be half the length of a frame), see [Nyquist rate](https://en.m.wikipedia.org/wiki/Nyquist_rate).
```

Comment:
Consider changing to https://en.wikipedia.org/wiki/Nyquist_rate. The desktop site redirects to the mobile site for mobile users but the mobile site doesn't redirect to the desktop site for desktop users.

Apart from that, I don't oppose this, especially given how far mobile devices have come since this page was originally written. I'm already using a smaller step than 1/60 to avoid jitter even on 60Hz displays.

Other ideas to consider include interpolating the simulation, or basing your step size on the user's refresh rate (`Gdx.graphics.getDisplayMode().refreshRate * 2` but avoid doing that every frame) if you don't need deterministic results.

Reply:
I removed the reference to the wiki article altogether : P